jupyter_sphinx/execute.py

```diff
@@ -437,6 +565,31 @@ def sphinx_abs_dir(env):
     )
 
 
+def builder_inited(app):
+    require_url = app.config.jupyter_sphinx_require_url
+    # 3 cases
+    # case 1: ipywidgets 6, only embed url
+    # case 2: ipywidgets 7, with require
+    # case 3: ipywidgets 7, no require
+    # (ipywidgets6 with require is not supported, require_url is ignored)
+    if has_embed:
```

Comment:
I think there is no right approach here, it could be that the html-manager you want does not correspond to any of the versions quoted in any package installed (imagine a patch release).
I'm in favor for bumping up the required ipywidgets version to 7, drop the has_embed, and consider what ipywidgets gives us a good default, and otherwise use `app.config.jupyter_sphinx_embed_url`
@SylvainCorlay do you agree?

Reply:
I wonder if the configuration option should be called `app.config.jupyter_sphinx_widget_embed_url` (it's only for embedding widgets and not anything else).